pkg/cli/command_test.go

```diff
@@ -244,14 +246,19 @@ func TestListCommandOutput(t *testing.T) {
 	}
 	for _, tt := range cases {
 		tt := tt
+		specialOutFileFlag := false
 		testName, expectedOutputFileName := getListCmdTestNameAndExpectedOutputFile(tt.dirName, tt.focusWorkload, tt.format)
+		if tt.specialExpectedOutFile != "" {
+			expectedOutputFileName = tt.specialExpectedOutFile
+			specialOutFileFlag = true
+		}
 		t.Run(testName, func(t *testing.T) {
 			args := []string{"list", "--dirpath", filepath.Join(testutils.GetTestsDirWithDepth(currentDirDepth), tt.dirName)}
 			args = append(args, addCmdOptionalArgs(tt.format, tt.outputFile, tt.focusWorkload)...)
 			actualOut, err := buildAndExecuteCommand(args)
 			require.Nil(t, err, "test: %q", testName)
 			testutils.CheckActualVsExpectedOutputMatch(t, tt.dirName, expectedOutputFileName, actualOut, testInfo(testName), tt.outputFile,
-				currentDirDepth)
+				currentDirDepth, specialOutFileFlag)
```

Comment:
I think we should avoid having "special flags", as this will be more confusing and difficult to maintain in the long term. Can you instead change the location for all expected output files, so that we have a uniform handling, and so that those output files would not interfere with the analysis results?

Reply:
done